.vscode/settings.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
What's this file for?

I'm not too familiar with all the config options.

Reply:
Sorry, that's my miss.
It should go into .gitignore.